src/Jenkins.ts

```diff
@@ -88,7 +88,12 @@ export class Jenkins {
       let result: JenkinsStatus;
       
       request
-        .get(url + '/api/json')
+        .get(url + '/api/json', {
+          'auth': {
```

Comment:
I secured my 1.647 instance and tested.  All scenarios perform as expected, whether providing a password or a token.